### DISCUSSION TITLE
Automation

### DISCUSSION THREAD

**Discussion by Joe-githubby** [music-assistant/support#2214] [Q&A] [answered]
Good day,

Is there a way to set up an automation to automatically start up a radio station on a speaker at a set time every day?

I have tried setting this up but the options don't come up.

**Comment by OzGav** [comment, marked as answer]
Really...

https://www.home-assistant.io/docs/automation/trigger/#time-trigger

https://music-assistant.io/faq/how-to/

**Reply by Joe-githubby** [reply]
Yes really. 

![Screenshot_20240508_141700_Home Assistant.jpg](https://github.com/music-assistant/hass-music-assistant/assets/71907834/d8e0fcb0-0fbf-4233-8f29-910169515f14)

**Reply by OzGav** [reply]
Whats with the picture? Why aren't you showing an image of the automation editor and what you have tried and how it isn't working?

**Reply by Joe-githubby** [reply]
When selecting media, music assistant is not an option.

Don't stress. You don't seem like too nice of a person to deal with in any event. I'll figure something out.

Thanks and sorry to bother you.

**Reply by agdamsbo** [reply]
> When selecting media, music assistant is not an option.

From your description and the picture, @Joe-githubby , it appears that you may be missing the Music Assistant integration. Try have a look at the installation notes: https://music-assistant.io/installation/

If this does not solve your problem, please provide a few more details 👍
